gradle/buildsrc.libs.versions.toml

```diff
@@ -1,9 +1,10 @@
 [versions]
-asciidoctorj = '4.0.2'
-asset-pipeline-gradle = '4.4.0'
-grails-gradle-plugin = '6.1.2'
+asciidoctorj = '4.0.3'
+asset-pipeline-gradle = '5.0.0'
```

Comment:
There is a `5.0.1` version released now.

Reply:
Done.